Content.Client/Clothing/Systems/ChameleonClothingSystem.cs

```diff
@@ -1,4 +1,5 @@
-﻿using System.Linq;
+using System.Linq;
```

Comment:
Linq jumpscare

Reply:
I didn't add this Linq! D: It was there before my PR.

It's used in this line:
`private static readonly SlotFlags[] Slots = Enum.GetValues<SlotFlags>().Except(IgnoredSlots).ToArray();`

I don't know how one would do this functionality without Linq though...